### PR TITLE
Dfr 778 change in hours for ctsc

### PR DESCRIPTION
**Pull request by MindGames2015**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: common/content.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "backLink": "Back",
     "isThereAProblemWithThisPage": "Contact us for help",
     "phoneTitle": "Telephone",
-    "phoneToCallIfProblems": "0300 303 0642<br>Monday to Friday, 8am to 8pm, Saturday 8am to 2pm",
+    "phoneToCallIfProblems": "0300 303 0642<br>Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.",
     "responseTime": "We aim to get back to you within 5 days",
```
Comment (arun-hmcts) [inline common/content.json:9]: Its minor comments, Please check with Ellen 
Webchat open 8-8.

Webchat open 8am to 8 pm

File: common/content.json [hunk 2]
```diff
@@ -14,9 +14,9 @@
     "chatWithAnAgent": "Chat online with an agent",
     "noAgentsAvailable": "No agents are available, please try again later.",
     "allAgentsBusy": "All our web chat agents are busy helping other people. Please try again later or contact us using one of the ways above.",
-    "chatClosed": "Web chat is now closed.\\nOpening hours are Monday to Friday, 8am to 8pm, Saturday 8am to 2pm.",
+    "chatClosed": "Web chat is now closed.\\nOpening hours are Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.",
```
Comment (arun-hmcts) [inline common/content.json:17]: same as above

File: steps/co-respondent/cr-done/CrDone.content.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     "help": "Getting help",
     "careOfAbbreviation": "c/o",
     "phone": "Phone",
-    "divorceCenterHours": "Monday to Friday, 8am to 8pm, Saturday 8am to 2pm",
+    "divorceCenterHours": "Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.",
```
Comment (arun-hmcts) [inline steps/co-respondent/cr-done/CrDone.content.json:22]: Webchat open 8am - 8pm

File: steps/co-respondent/cr-progress-bar/CrProgressBar.content.json [hunk 4]
```diff
@@ -66,7 +66,7 @@
     "help": "Getting help",
     "email": "Email",
     "phone": "Phone",
-    "divorceCenterHours": "(Monday to Friday, 8am to 8pm, Saturday 8am to 2pm)",
+    "divorceCenterHours": "(Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.)",
```
Comment (arun-hmcts) [inline steps/co-respondent/cr-progress-bar/CrProgressBar.content.json:69]: Webchat open 8am - 8 pm

File: steps/co-respondent/cr-respond/CrRespond.content.json [hunk 5]
```diff
@@ -17,7 +17,7 @@
     "help": "Getting help",
     "careOfAbbreviation": "c/o",
     "phone": "Phone",
-    "divorceCenterHours": "Monday to Friday, 8am to 8pm, Saturday 8am to 2pm",
+    "divorceCenterHours": "Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.",
```
Comment (arun-hmcts) [inline steps/co-respondent/cr-respond/CrRespond.content.json:20]: Webchat open 8am - 8 pm

File: steps/divorce-application-processing/DivorceApplicationProcessing.content.json [hunk 6]
```diff
@@ -6,7 +6,7 @@
     "email": "Email",
     "phone": "Phone",
     "divorceEmail": "divorcecase@justice.gov.uk",
-    "openTimes": "(Monday to Friday, 8am to 8pm, Saturday 8am to 2pm)",
+    "openTimes": "(Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.)",
```
Comment (arun-hmcts) [inline steps/divorce-application-processing/DivorceApplicationProcessing.content.json:9]: Webchat open 8am to 8pm

File: steps/end/End.content.json [hunk 7]
```diff
@@ -4,7 +4,7 @@
     "progressNotSaved": "Your progress has not been saved.",
     "signInToReturn": "<a class=\"govuk-link\" href=\"/\" aria-label=\"Sign in link\">Sign in</a> to return to your response.",
     "ifYouNeedHelp": "If you need help with this service you can call or email us.",
-    "phoneToCallIfProblems": "Phone: 0300 303 0642 (Monday to Friday, 8am to 8pm, Saturday 8am to 2pm)",
+    "phoneToCallIfProblems": "Phone: 0300 303 0642 (Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.)",
```
Comment (arun-hmcts) [inline steps/end/End.content.json:7]: Webchat open 8am - 8pm

File: steps/respondent/respond/Respond.content.json [hunk 8]
```diff
@@ -20,7 +20,7 @@
     "help": "Getting help",
     "careOfAbbreviation": "c/o",
     "phone": "Phone",
-    "divorceCenterHours": "Monday to Friday, 8am to 8pm, Saturday 8am to 2pm",
+    "divorceCenterHours": "Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.",
```
Comment (arun-hmcts) [inline steps/respondent/respond/Respond.content.json:23]: Webchat open 8am - 8pm

File: views/errors/error-content.json [hunk 9]
```diff
@@ -2,7 +2,7 @@
   "serviceName": "Respond to a divorce application",
   "isThereAProblemWithThisPage": "Is there a problem with this page?",
   "isThereAProblemWithThisPageParagraph": "You can call or email us if you’re having problems with this service.",
-  "isThereAProblemWithThisPagePhone": "Phone: 0300 303 0642 (Monday to Friday, 8am to 8pm, Saturday 8am to 2pm)",
+  "isThereAProblemWithThisPagePhone": "Phone: 0300 303 0642 (Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.)",
```
Comment (arun-hmcts) [inline views/errors/error-content.json:5]: Webchat open 8am - 8pm

File: steps/accessibility-statement/AccessibilityStatement.content.json [hunk 10]
```diff
@@ -20,7 +20,7 @@
     "whatToDoIfYouCanAccessParts": "What to do if you can’t access parts of this website",
     "needMoreInformation": "If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:",
     "email": "email <a href=\"mailto:HMCTSforms@justice.gov.uk\" class=\"govuk-link\" aria-label=\"This link will open in a new email to HMCTSforms@justice.gov.uk\">HMCTSforms@justice.gov.uk</a>",
-    "phone": "call 0300 303 0642 (Monday to Friday, 8am to 8pm, Saturday 8am to 2pm)",
+    "phone": "call 0300 303 0642 (Monday, Wednesday and Friday -phones are open 8am-6pm and Webchat open 8-8. <br>Tuesday and Thursday - Webchat and phones are open 8am-8pm. <br>Saturdays - Webchat and phones are open 8am-2pm.)",
```
Comment (arun-hmcts) [inline steps/accessibility-statement/AccessibilityStatement.content.json:23]: Webchat open 8am - 8pm

File: common/content.json (hunk 1 above)
Comment (MindGames2015) [reply common/content.json:9]: Thanks Arun I'll check with Ellen

File: steps/co-respondent/cr-progress-bar/CrProgressBar.content.json (hunk 4 above)
Comment (MindGames2015) [reply steps/co-respondent/cr-progress-bar/CrProgressBar.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

Comment (MindGames2015) [reply steps/co-respondent/cr-progress-bar/CrProgressBar.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

Comment (MindGames2015) [reply steps/co-respondent/cr-progress-bar/CrProgressBar.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: steps/co-respondent/cr-respond/CrRespond.content.json (hunk 5 above)
Comment (MindGames2015) [reply steps/co-respondent/cr-respond/CrRespond.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: steps/accessibility-statement/AccessibilityStatement.content.json (hunk 10 above)
Comment (MindGames2015) [reply steps/accessibility-statement/AccessibilityStatement.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: views/errors/error-content.json (hunk 9 above)
Comment (MindGames2015) [reply views/errors/error-content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: common/content.json (hunk 1 above)
Comment (MindGames2015) [reply common/content.json:9]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: common/content.json (hunk 2 above)
Comment (MindGames2015) [reply common/content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: steps/co-respondent/cr-done/CrDone.content.json (hunk 3 above)
Comment (MindGames2015) [reply steps/co-respondent/cr-done/CrDone.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: steps/end/End.content.json (hunk 7 above)
Comment (MindGames2015) [reply steps/end/End.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: steps/divorce-application-processing/DivorceApplicationProcessing.content.json (hunk 6 above)
Comment (MindGames2015) [reply steps/divorce-application-processing/DivorceApplicationProcessing.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

File: steps/respondent/respond/Respond.content.json (hunk 8 above)
Comment (MindGames2015) [reply steps/respondent/respond/Respond.content.json]: Updated line 'Webchat open 8-8', to 'Webchat open 8am-8pm'

**Pull request closed without merging**